script/Arcueid/Day 7/07_00_ARC07_1_1.txt

```diff
@@ -6,12 +6,12 @@ It's currently 7:00 PM.
 [sha:0ae30c8b668323e38414fbd46198de0cf1a7393a]{
 -- Page 10004, Offset 16897.
 -- もともと車通りの少ない国道だが、夜ともなれば完全に<人気|ひとけ>は途絶え、周囲は静まりかえっている。
-It was a highway with very little traffic to begin with, but as the night comes, all signs of life come to a halt and the surroundings fall silent.
+It had been a highway with very little traffic to begin with, but as the night came, all signs of life came to a halt and the surroundings fell silent.
```

Comment:
Revert, tense is correct. Also "had been" a highway makes it sounds like it is no longer a highway.